src/Ledger/TokenAlgebra.lagda

```diff
@@ -9,27 +9,36 @@ open import Algebra using (CommutativeMonoid)
 open import Algebra.Morphism
 open import Data.Integer
 open import Data.Nat.Properties using (+-0-commutativeMonoid)
+\end{code}
 
+\begin{figure*}
+\begin{code}
 record TokenAlgebra : Set₁ where
   field PolicyId : Set
         Value-CommutativeMonoid : CommutativeMonoid 0ℓ 0ℓ
 
   MemoryEstimate = ℕ
-
+\end{code}
+\begin{code}[hide]
   open CommutativeMonoid Value-CommutativeMonoid using (_≈_; ε)
        renaming (Carrier to Value; refl to reflᵛ; _∙_ to _+ᵛ_) public
-
-  field coin     : Value → Coin
-        inject   : Coin → Value
-        policies : Value → ℙ PolicyId
-        size     : Value → MemoryEstimate
+\end{code}
+\begin{code}
+  field coin      : Value → Coin
+        inject    : Coin → Value
+        policies  : Value → ℙ PolicyId
+        size      : Value → MemoryEstimate
         _≥ᵗ_      : Value → Value → Set
```

Comment:
Hmmm... I thought we already changed `_≥ᵗ_` to the less or equal symbol 🤔  Maybe not.  Oh well, hopefully this will get picked up in a later PR... but not important.